bin/dry-run.rb

```diff
@@ -799,8 +799,8 @@ def security_fix?(dependency)
 StackProf.results("tmp/stackprof-#{Time.now.strftime('%Y-%m-%d-%H:%M')}.dump") if $options[:profile]
 
 puts "🌍 Total requests made: '#{$network_trace_count}'"
-package_manager = fetcher.package_manager_version
-puts "🎈 Package manager version log: #{package_manager}" unless package_manager.nil?
+e_versions = fetcher.ecosystem_versions
+puts "🎈 Ecosystem Versions log: #{e_versions}" unless e_versions.nil?
```

Comment:
```suggestion
ecosystem_versions = fetcher.ecosystem_versions
puts "🎈 Ecosystem Versions log: #{ecosystem_versions}" unless ecosystem_versions.nil?
```

Style nit: Variable contractions always feel a little un-Ruby to me